src/views/token-swap/components/selected-quote/selected-quote.style.js

```diff
@@ -61,16 +61,6 @@ const useSelectedQuoteStyles = createUseStyles((theme) => ({
     color: theme.palette.grey.main,
     fontWeight: theme.fontWeights.bold,
   },
-  reward: {
-    borderTop: `1px solid ${theme.palette.grey.veryLight}`,
```

Comment:
Token swap use different rewards. They are not related to this issue

Reply:
Thx! Reverted!